doc/autoscale/experiment/mnist/train.py

```diff
@@ -0,0 +1,163 @@
+from PIL import Image
+import numpy as np
+import paddle.v2 as paddle
+import paddle.v2.dataset.common as common
+import os
+import sys
+import glob
+import pickle
+
+
+# NOTE: must change this to your own username on paddlecloud.
+USERNAME = "yanxu05@baidu.com"
+DC = os.getenv("PADDLE_CLOUD_CURRENT_DATACENTER")
+#common.DATA_HOME = "/pfs/%s/home/%s" % (DC, USERNAME)
```

Comment:
Is this unintentionally commented out?

Reply:
Done, delete the unnecessary commented code.